src/Illuminate/Database/Migrations/stubs/migration.create.stub

```diff
@@ -11,7 +11,7 @@ return new class extends Migration
      *
      * @return void
      */
-    public function up()
+    public function up(): void
```

Comment:
There's doctype already @andrejsmelnikovs

Reply:
Doctype doesn't throw an error, if returned value is not of the correct type.

Reply:
That's what a static analyzer is for. :)